CONTRIBUTING.md

```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
 1. `yarn install`
 1. `yarn build`
 
+## Running tests
+
+1. Run unit tests: `yarn test`
+1. Run unit tests and show test coverage info for all tests: `yarn test --coverage --watchAll`
```

Comment:
I don't think this is necessary - it's more for a maintainer than a contributor.

Reply:
I was thinking maybe adding these would encourage contributors to pay attention to / add their own unit tests when submitting PRs. The commands are pretty standard but for some newer contributors they might not be familiar with reading the scripts section of package.json and the coverage watchAll feature of jest.